File: reactive-dashboard/src/app/entity/stock.ts
export class Stock { 
    id: number; 
    alias: string; 
    name: string; 
    lastDayMarketPrice: number; 
    currentMarketPrice: number; 
    low52Week: number; 
    high52Week: number; 
    
    
    constructor(id: number, alias: string, name: string, lastDayMarketPrice: number, 
        currentMarketPrice: number, low52Week: number, high52Week: number) 
    { 
        this.id = id; 
        this.alias = alias; 
        this.name = name; 
        this.lastDayMarketPrice = lastDayMarketPrice; 
        this.currentMarketPrice = currentMarketPrice; 
        this.low52Week = low52Week; 
        this.high52Week = high52Week; 
    } 
}

File: ReactiveSpringboot/src/main/java/com/reactive/programming/ReactiveSpringboot/entity/Stock.java
package com.reactive.programming.ReactiveSpringboot.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@AllArgsConstructor
public class Stock {
	private int id;
	private String alias;
	private String name;
	private double lastDayMarketPrice;
	private double currentMarketPrice;
	private double low52Week;
	private double high52Week;

}
